cmake/alpakaCommon.cmake

```diff
@@ -655,8 +672,6 @@ if(alpaka_ACC_SYCL_ENABLE)
         find_package(oneDPL REQUIRED)
         target_link_libraries(alpaka INTERFACE oneDPL)
     endif()
-
-    alpaka_set_compiler_options(DEVICE target alpaka "-fsycl-unnamed-lambda") # Compiler default but made explicit here
```

Comment:
This was removed because it is already set above in line 657. Looks like a faulty merge that we have overlooked previously.